README.md

```diff
@@ -1,26 +1,32 @@
+
+## Supported hardware:
+* Sonoff ZBDongle-E by ITead 
+* Sonoff iHost by ITead (uses the same firmware as Sonoff ZBDongle-E )
 * EasyIot ZB-GW04 v1.1 and v1.2
-* Elelabs ELU013 and Raspi Shield
 * Smlight SLZB-07 (may require unlocked [bootloader](https://github.com/darkxst/silabs-firmware-builder/raw/main/firmware_builds/slzb-07/BTL_SLZB07.gbl) first)
 * Sparkfun Things Matter MGM240P (requires [bootloader](https://github.com/darkxst/silabs-firmware-builder/blob/main/firmware_builds/mgm240p/bootloader-uart-xmodem_NCP.hex) to be flashed first using Silabs [Simplicity Commander](https://community.silabs.com/s/article/simplicity-commander?language=en_US))
+* Elelabs Zigbee USB Adapter ELU013 (based on EFR32MG13P)
+* Elelabs Zigbee Raspberry Pi Shield ELR023 (based on EFR32MG13P)
+
+## Different firmwware variants
 
-It uses the Silicon Labs Gecko SDK and proprietary Silicon Labs tools such as
-the Silicon Labs Configurator (slc) and the Simplicity Commander standalone
-utility. This is a fork of the [NabuCasa](https://github.com/NabuCasa/silabs-firmware-builder) silabs firmware builder, adding support for additional hardware.
+Three network protocol application firmware variants are available:
 
-Three firmware variants are available:
-* EmberZNet NCP for Zigbee-only environments, for direct use with Zigbee2MQTT, ZHA or Zigpy or other Zigbee stacks
-* RCP Multi-PAN firmware for concurrent communication over Zigbee and Thread
-* OpenThread RCP firmware (experimental) Thread-only
+* **EmberZNet NCP** = Zigbee NCP (Network Co-Processor) is used as dedicated Zigbee Coordinator for Zigbee-only environments, for direct use with Zigbee2MQTT, Home Asssiatnt's ZHA integration, other Zigpy based Zigbee Gateway implementations, or other Zigbee gateways/frameworks that support the EZSP (EmberZNet Serial Protocol) interface.
+* **OpenThread RCP** firmware (experimental) = This [Thread](https://en.wikipedia.org/wiki/Thread_(network_protocol)) RCP (Radio Co-Processor) is used directy as dedicated Thread Border Router in Thread-only environments, used for [OpenThread Border Router add-on](https://github.com/home-assistant/addons/blob/master/openthread_border_router/DOCS.md) or wpantund.
+* **RCP Multi-PAN** (no longer recommended) = Multiprotocol firmware for concurrent communication over Zigbee and Thread via Home Assistant [SiliconLabs Multiprotocol add-on](https://github.com/home-assistant/addons/blob/master/silabs-multiprotocol/DOCS.md).
+  * Note that RCP MultiPAN in multiprotocol mode is no longer recommended as proven to not be stable with medium-sized networks or larger when using Zigbee and Thread simulationaly on the same radio adapter, though it can still be used when starting out with only a few devices if plan to migrate to separate dedicated radio adapters with Zigbee NCP  and Thread RCP firmware once added more devices (suggest not using more more than around ~20 devices on a single RCP Multi-PAN adapter that is used in Mmlti-protocol mode).
```

Comment:
I would probably discourage new users going down this path at all. The instabilities are not really tied to network size, but more just a general regression in the stability of the upstream releases.